src/screens/ChannelListScreenV2/IncognitoEmptyChat/index.tsx

```diff
@@ -31,9 +32,14 @@ const IncognitoButton = ({title, subtitle, onPress}) => {
   );
 };
 
-const IncognitoEmptyChat = ({totalChannel}) => {
-  const {goToContactScreen} = useAnonymousChannelListScreenHook();
+const IncognitoEmptyChat = () => {
+  const {channels: anonChannels, goToContactScreen} = useAnonymousChannelListScreenHook();
   const navigation = useNavigation();
+  let totalChannel = anonChannels.length;
+  const storedTotalChannel = StorageUtils.totalAnonChannels.get();
```

Comment:
mas @GarryLaly please move this to useEffect.
StorageUtils is heavy load operation, if it is on component scope like this it will be called every render which will impact on the performance